components/expenses/ExpenseFormPayeeStep.js

```diff
@@ -33,6 +36,10 @@ const msg = defineMessages({
     id: `ExpenseForm.payeeLabel`,
     defaultMessage: 'Who is being paid for this expense?',
   },
+  payeeLegalName: {
+    id: `ExpenseForm.payeeLegalName`,
+    defaultMessage: 'Legal name of payee {tooltip}',
```

Comment:
I think there's value in removing `{tooltip}` from the string (it's not necessary, and doing so will make contributor's life easier). Also, I don't know if it's useful to repeat `of payee` here - we're in the `Payee information` section. Same way we don't write `Address of payee`.

I can propose:

```jsx
label=(
  <React.Fragment>
    <FormattedMessage defaultMessage="Legal name" />
    &nbsp;
    <StyledTooltip ...><Icon /></StyledTooltip>
  </React.Fragment>
)
```

Reply:
@Betree : Done, I've refactored it to remove the `{tooltip}` from the string. Btw, can you explain why it makes the contributors life easier? 🤔 I couldn't figure that one out. 😉

Reply:
1. Because the string is more easily re-usable. We know for a fact that the `Payee name` string is used in many different places, while `Legal name of payee {tooltip}` was going to be used just there. Fewer strings to translate = less work.
2. When you need to translate the strings on https://crowdin.com, having arguments in the string force you to understand the context: what's going do be displayed with that argument? Does it impact the order? When there's no argument, and for simple terms like that, you can usually rely on the auto-translate feature of Crowdin wich makes it very easy to batch translations.

Reply:
Thanks much for the explanation. That makes perfect sense. 👍🏽